setup.py

```diff
@@ -34,6 +34,7 @@
     "enforce-typing==1.0.0.post1",
     "json-sempai==0.4.0",
     "eciespy",
+    "huggingface_hub"
```

Comment:
please set the dependency version. Whatever is non-grounded now is probably due to dependency resolving issues, but if possible let's have the dependency in clear. (Or a comment explaining why it's not)